tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -81,6 +81,68 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-efx-generic-4ch.bin,\
 EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
 EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 
+# With 16 kHz DMIC1
+"sof-hda-generic\;sof-hda-generic-4ch-48k-16k\;\
+HDA_CONFIG=mix,DEEPBUFFER_FW_DMA_MS=100,\
```

Comment:
`DEEPBUFFER_FW_DMA_MS=100`  can be dropped, it is the default

Reply:
Done!